zdm/pcosmic.py

```diff
@@ -82,15 +83,18 @@ def make_C0_grid(zeds,F):
 		C0s[i]=iterate_C0(z,F)
 	return C0s
 
-def get_mean_DM(zeds):
+def get_mean_DM(zeds,H0=cos.DEF_H0):
 	""" Gets mean average z to which can be applied deltas """
 	
 	zmax=zeds[-1]
 	nz=zeds.size
 	sys.path.insert(1, '/Users/cjames/CRAFT/FRB_library/FRB-master/')
```

Comment:
this hard-coding is dodgy at best. but maybe it is ignored..

Reply:
This is so that 67.74 isn't hard-coded everywhere and the default value set in the cosmology.py file is taken. Wasn't sure how else to do this.

Reply:
This seems fine to me - but note my comment on how to get DEF_H0 in the first place!